cairo-src/evm/utils.cairo

```diff
@@ -25,6 +25,15 @@ func ceil_div{range_check_ptr}(a, b) -> (res):
     end
 end
 
+func memory_expansion_for_range{range_check_ptr}(s, f, l) -> (res):
```

Comment:
Perhaps, `update_msize` is a more suitable name? Together with more descriptive parameter names, it would make the function crystally clear